examples/raw.rs

```diff
@@ -1,26 +1,18 @@
-// Copyright 2018 The TiKV Project Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
+
+#![feature(async_await, await_macro)]
```

Comment:
Do we have an ETA on stable?

Reply:
August 15th. Note that I only use unstable features in the examples, the library itself is all stable (well, technically it needs beta, but will all be stable at the next release).

Reply:
Ok. I think it's fine if we make this nightly-only for now as long as we are on the road to stable.